src/sections/Company/Legal/privacy/index.js

```diff
@@ -112,7 +112,7 @@ const Conduct = () => {
             </div>
             <p>We welcome your comments or questions about this privacy policy. You may also contact us as follows:
             </p>
-            <p class="address">
+            <p className="address">
```

Comment:
Ok. Can you checkout to the master branch and fetch the latest upstream, and check if confirm if we have this or not. I think it has been updated recently after you have opened this PR, so this change might no be needed.

Reply:
I did that and there's no changes

Reply:
Interesting, I checked the master code and it's fixed.
Check this: https://github.com/layer5io/layer5/blob/7ef1455430b873b3c17ffc17654cf76cd34ca43b/src/sections/Company/Legal/privacy/index.js#L115

Reply:
Okay, let's ignore that for the time being considering there have been a lot of too and fro now, I will take that in and not de-motivate you to contribute more 😄